packages/strapi-generate-new/lib/create-project.js

```diff
@@ -0,0 +1,132 @@
+const { trackUsage } = require('./utils/usage');
+const packageJSON = require('./resources/json/package.json');
+const databaseJSON = require('./resources/json/database.json.js');
+
+module.exports = async function createProject(
+  scope,
+  { connection, dependencies }
+) {
+  console.log('Creating files.');
+
+  const { rootPath } = scope;
+  const resources = join(__dirname, 'resources');
+
+  try {
+    // copy files
+    await fse.copy(join(resources, 'files'), rootPath);
+
+    // copy templates
+    await fse.writeJSON(
+      join(rootPath, 'package.json'),
+      packageJSON({
+        strapiDependencies: scope.strapiDependencies,
+        additionalsDependencies: dependencies,
+        strapiVersion: scope.strapiVersion,
+        projectName: scope.name,
+        uuid: scope.uuid,
+      }),
+      {
+        spaces: 2,
+      }
+    );
+
+    // ensure node_modules is created
+    await fse.ensureDir(join(rootPath, 'node_modules'));
+
+    await Promise.all(
+      ['development', 'staging', 'production'].map(env => {
+        return fse.writeJSON(
+          join(rootPath, `config/environments/${env}/database.json`),
+          databaseJSON({
+            connection,
+            env,
+          }),
+          { spaces: 2 }
+        );
+      })
+    );
+  } catch (err) {
+    await fse.remove(scope.rootPath);
+    throw err;
+  }
+
+  const installPrefix = chalk.yellow('Installing dependencies:');
+  const loader = ora(installPrefix).start();
+
+  const logInstall = (chunk = '') => {
+    loader.text = `${installPrefix} ${chunk
+      .toString()
+      .split('\n')
+      .join(' ')}`;
+  };
+
+  try {
+    const runner = runInstall(scope);
+
+    runner.stdout.on('data', logInstall);
+    runner.stderr.on('data', logInstall);
+
+    await runner;
+
+    loader.stop();
+    console.log(`Dependencies installed ${chalk.green('successfully')}.`);
+  } catch (error) {
+    loader.stop();
+    await trackUsage({
+      event: 'didNotInstallProjectDependencies',
+      scope,
+      error,
+    });
+
+    stopProcess(
+      `${chalk.red(
+        'Error'
+      )} while installing dependencies:\n${error.stderr.trim()}`
+    );
+  }
+
+  await trackUsage({ event: 'didCreateProject', scope });
+
+  console.log();
+  console.log(`Your application was created at ${chalk.green(rootPath)}.\n`);
+
+  const cmd = chalk.cyan(scope.useYarn ? 'yarn' : 'npm run');
+
+  console.log('Available commands in your project:');
+  console.log();
+  console.log(`  ${cmd} develop`);
+  console.log('  Starts Strapi in watch mode.');
+  console.log();
+  console.log(`  ${cmd} start`);
+  console.log('  Starts Strapi without watch mode.');
+  console.log();
+  console.log(`  ${cmd} build`);
+  console.log('  Builds Strapi admin panel.');
+  console.log();
+  console.log('You can start by doing:');
+  console.log();
+  console.log(`  ${chalk.cyan('cd')} ${rootPath}`);
+  console.log(`  ${cmd} develop`);
+  console.log();
```

Comment:
Did we remove the strapi console command I recall that this one was usefull

Reply:
Agreed @soupette `strapi console` is extremely useful especially with most of the internal API not being completely documented.

Reply:
Nope it wasn't deleted I'm just logging the 3 main commands before the user starts using strapi. I don't want them to have to read 10 commands before starting. 

I'll add sth about listing the available commands

Reply:
Yes you're right, anyway they can still run `strapi --help`